### DISCUSSION TITLE
Unable to run Local search due to unavailable "create_final_covariates" file

### DISCUSSION THREAD

**Discussion by mystvearn** [microsoft/graphrag#459] [Q&A]
I'm following the tutorial on graphRAG website to run through global search and local search example. The global search one ran smoothly but I had problem with local search due to missing "create_final_covariates.parquet" file. 

I assume this is error from the graphRAG package itself. Please help to update the package or propose a way to fix it

**Comment by rohit-metrum-ai** [comment]
you can check this https://github.com/microsoft/graphrag/issues/508#issuecomment-2224095665
